### PR TITLE
Make package naming uniform

### PR DESCRIPTION
**Pull request by lambdista**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,33 +30,24 @@ Clone the repository.
...
+| Chapter 11 - Java Compatibility | src/main/scala/com/wrox/professionalscala/ch11 |
 
 ## Usage
 
-### Chapters 1, 2, 4, 7, 11 and some of Chapter 3
+### Chapters 1, 2, 3, 4, 7, 8, 9, 10 and 11
 
 Run the examples using `sbt` on the  command line (sbt run  will do the trick).
 
-### Chapters 9, 10 and some of Chapter 3
```
Comment (janekdb) [inline README.md:51]: This is not related to package naming. Was this intended to be included? If the PR includes only the package naming changes we can merge it without discussion. Suggest to move the running instructions to a new PR to allow the package naming change to be merged.

Comment (lambdista) [inline README.md:51]: Was meant to be included in the sense that I removed the `runMain` part and left only the `sbt run` part which seems to be OK for everyone.

Comment (janekdb) [inline README.md:51]: @lambdista - Please rebase this commit and push. It has conflicts.
